pkg/bot/slack.go

```diff
@@ -67,8 +67,13 @@ type slackMessage struct {
 
 // NewSlack creates a new Slack instance.
 func NewSlack(log logrus.FieldLogger, cfg config.Slack, executorFactory ExecutorFactory, reporter FatalErrorAnalyticsReporter) (*Slack, error) {
-	client := slack.New(cfg.Token, slack.OptionAppLevelToken(SlackAppLevelToken))
-
```

Comment:
Please introduce some logging around that and also validation.

Here I create my proposition:  https://github.com/kubeshop/botkube/commit/5453531488df238a8a14f2a65b736005c0c166cd



I was not able to do the validation via tags, so I registered a dedicated function. IMO we should move to JSON Schema validation at some point.

Reply:
sure